### PR TITLE
Implement #5826: Add command to show map data counts and limits

### PR DESCRIPTION
**Pull request by oli414**
This PR adds the `show_limits` command to show the map-data counts and limits. As suggested in https://github.com/OpenRCT2/OpenRCT2/issues/5826.

objlimits

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I prefer `show_limits`. None of those are objects in the usual RCT2 sense.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@
 - Feature: [#6292] Allow building queue lines in the Scenario Editor.
 - Feature: [#6325] Allow using g1.dat from RCT Classic.
 - Feature: [#6353] Show custom RCT1 scenarios in New Scenario window.
+- Feature: [#6368] Add the show_limits command to show map data counts and limits.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: Put the issue number here, not the PR number (unless there isn't one).

File: src/openrct2/interface/console.c [hunk 2]
```diff
@@ -1289,6 +1336,7 @@ console_command console_command_table[] = {
     { "rides", cc_rides, "Ride management.", "rides <subcommand>" },
     { "staff", cc_staff, "Staff management.", "staff <subcommand>"},
     { "remove_unused_objects", cc_remove_unused_objects, "Removes all the unused objects from the object selection.", "remove_unused_objects" },
+    { "show_limits", cc_show_limits, "Shows the map data counts and limits..", "show_limits" },
```
Comment (janisozaur) [inline src/openrct2/interface/console.c]: duplicated full stop

**Comment by Gymnasiast** [comment]
That last commit was a case of `[ci skip]` ;-)

**Comment by janisozaur** [comment]
I think it would be useful to also show percentage of each entry, but I'm not too fussed about it. LGTM.

**Comment by Gymnasiast** [comment]
Merged! Could you add the command to the Wiki page?

**Pull request merged**